api/v1alpha1/operator_types.go

```diff
@@ -78,6 +80,8 @@ func init() {
 
 // OperatorStatus defines the observed state of Operator
 type OperatorStatus struct {
+	// +optional
+	InstalledBundleSource string `json:"installedBundleSource,omitempty"`
```

Comment:
Nit: I'm assuming this has a relation to the field below in that they both point to `Bundle` resources on cluster (please correct me if I am wrong). To keep the naming similar I would prefer the name `InstalledBundleResource`

Reply:
Alternatively, changing the field `ResolvedBundleResource` --> `ResolvedBundleSource` would also work

Reply:
Makes sense! Thanks! Let me change it to `InstalledBundleResource`.

Reply:
fixed